doc/compute-job-configuration.md

```diff
@@ -0,0 +1,390 @@
+
+The available operations are declared in the operations list using truth tables as follows:
+
+```json
+"operations": {
+	"AND":[],
+	"OR":[]
+}
+```
+
+Each operation consists of a JSON array used to describe a _truth table_. An example of such a _truth table_ is presented below:
+
+```json
+"operations": {
+	"AND": [
+	  { "A":"OK",       "B":"OK",       "X":"OK"       },
+	  { "A":"OK",       "B":"WARNING",  "X":"WARNING"  },
+	  { "A":"OK",       "B":"UNKNOWN",  "X":"UNKNOWN"  },
+	]
+}
+```
+
+Each element of the JSON array describes a row of the _truth table_ for example:
+
+```json
+{ "A":"OK", "B":"WARNING", "X":"WARNING"}
+```
+
+declares that in an algorithmic AND operation between two status states of *OK* and *WARNING* the result is *WARNING*
+
+In the ops file the user is able to declare any number of available monitoring states and any number of available custom operations on those states. The ARGO Compute Engine uses this information to create the corresponding _truth tables_ in memory.
+
+#### {TENANT_NAME}_{REPORT_ID}_cfg.json (per report)
+
+A _Report Configuration_ file is a JSON file that contains specific information needed during the computations such as grouping parameters, the name of the availability profile used and many more. If the tenant's name is `T1` and the report name is `ReportA` then the filename of the config file must be `T1_ReportA_cfg.json`
+
+The configuration file of the job contains mandatory and optional fields with rich information describing the paramaters of the specific job. Some important fields are:
+
+```json
+"tenant": "tenant_name"`
+"job": "job_name",
+"aprofile": "availability_profile_name",
+"egroup": "endpoint_group_type_name",
+"ggroup": "group_of_group_type_name",
+"weight": "weight_factor_type_name"
+```
+[](Comment: Are the egroup and ggroup correct? Needs review by @kaggis)
+
+In the above snippet we have declared the name of the tenant, the name of the job, the name of the specific availability profile used in the job. Also the type of endpoint grouping that will be used is declared here and the type of upper hierarchical grouping. Also if available here is declared the type of weight factor used for upper level A/R aggregations
+
+- `"tenant"`
+
+  This field is explicitly linked to the value of the tenant declaration of the global ar-compute-engine.conf file [link to description above](/#parameters-for-section-jobs)
+
+- `"job"`
+
+  This field is explicitly linked to the name of a job declared in the job_set variable of the global ar-compute-engine.conf file [link to description above](#parameters-for-section-jobs)
+
+- `"aprofile"`
+
+  This field is explicitly linked to one of the availability profile json files declared in the `/etc/ar_compute/` folder and they are described below [link to description further below](#availability-profile-per-tenant--per-job)
+
+- `"egroup"` [](Comment: Needs review by @kaggis)
```

Comment:
At the moment yes they are correct. This is how they are declared in the cfg files of the CE

this is going to change because they are expressed more correctly in the new reports definitions in the API.